serving/docs/lmi_new/user_guides/tnx_user_guide.md

```diff
@@ -0,0 +1,78 @@
+# Transformers-NeuronX Engine in LMI
+
+## Model Artifacts Structure
+
+LMI Transformers-NeuronX expect the model to be standard HuggingFace format for runtime compilation. For loading of compiled models, both Optimum compiled models and split-models with a separate `neff` cache (compiled models must be compiled with the same Neuron compiler version and model settings).
+The source of the model could be:
+
+- model_id string from huggingface
+- s3 url that store the artifact that follows the HuggingFace model repo structure, the Optimum compiled model repo structure, or a split model with a second url for the `neff` cache.
+- A local path that contains everything in a folder follows HuggingFace model repo structure, the Optimum compiled model repo structure, or a split model with a second directory for the `neff` cache.
+
+## Supported Model architecture
+
+LMI is shipping vLLM 0.2.7 with 0.26.0 containers, 
+so technically we support all LLM that [vLLM 0.2.7 support](https://github.com/vllm-project/vllm/tree/v0.2.7?tab=readme-ov-file#about).
+
+The model architectures that are tested daily for LMI Transformers-NeuronX (in CI):
+- LLAMA
+- Mistral
+- GPT-NeoX
+- GPT-J
+- Bloom
+- GPT2
+- OPT
+
+### Complete model set
+
+- BLOOM (`bigscience/bloom`, `bigscience/bloomz`, etc.)
+- GPT-2 (`gpt2`, `gpt2-xl`, etc.)
+- GPT BigCode (`bigcode/starcoder`, `bigcode/gpt_bigcode-santacoder`, etc.)
+- GPT-J (`EleutherAI/gpt-j-6b`, `nomic-ai/gpt4all-j`, etc.)
+- GPT-NeoX (`EleutherAI/gpt-neox-20b`, `databricks/dolly-v2-12b`, `stabilityai/stablelm-tuned-alpha-7b`, etc.)
+- LLaMA & LLaMA-2 (`meta-llama/Llama-2-70b-hf`, `lmsys/vicuna-13b-v1.3`, `young-geng/koala`, `openlm-research/open_llama_13b`, etc.)
+- Mistral (`mistralai/Mistral-7B-v0.1`, `mistralai/Mistral-7B-Instruct-v0.1`, etc.)
+- OPT (`facebook/opt-66b`, `facebook/opt-iml-max-30b`, etc.)
+
+We will add more model support for the future versions to have them tested. Please feel free to [file us an issue](https://github.com/deepjavalibrary/djl-serving/issues/new/choose) for more model coverage in CI.
+
+### Quantization
+
+Currently, we allow customer to use `option.quantize=static_int8` or `OPTION_QUANTIZE=static_int8` to load the model using `int8` weight quantization.
+
+## Quick Start Configurations
+
+Most of the LMI Transformers-NeuronX models use the following template:
```

Comment:
What cases would not?

Reply:
Added clarifying comments and link to the advanced configuration doc.